slsSupportLib/include/genericSocket.h

```diff
@@ -43,7 +43,7 @@ class sockaddr_in;
 #include <errno.h>
 #include <stdio.h>
 #include "logger.h"
-
+#include <atomic>
```

Comment:
minor. but you probably dont need this here.

Reply:
Well spotted, removed it now.